src/components/pages/CalculatorPage.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import Calculator from '../Calculator';
+
+const CalculatorPage = () => (
+  <>
+    <h3>Let Us Do Some Math!</h3>
+    <Calculator />
+  </>
+);
```

Comment:
Your design
![image](https://github.com/AleWaweru/Math-Magicians/assets/78251927/9fdde1bd-b27d-4804-affb-12845343f708)

Wireframe
![image](https://github.com/AleWaweru/Math-Magicians/assets/78251927/1dd200c8-bf11-49fe-8d00-68df63fb55cd)

As it is important that our general layout should match the given wireframes in the [project requirements](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_full_website.md).  👀


Kindly adjust the layout of the `Let Us Do Some Math!` and `Calculator` body according to the wireframe. 🛠👍

Reply:
done.